vuu-ui/packages/vuu-filters/src/filter-clause/ColumnPicker.tsx

```diff
@@ -10,6 +10,7 @@ export type ColumnPickerProps = Pick<
 > & {
   columns: ColumnDescriptor[];
   onSelect: (evt: SyntheticEvent, columnName: string) => void;
+  defaultDropdown?: ExpandoComboboxProps["defaultDropdown"];
```

Comment:
We show dropdown on focus. Salt used to have the same concept, the older DropDownBase used to have a `openOnFocus` prop, defaulted to true. The current version no longer supports it, but I think its a good descriptive name for the purpose of this prop. `defaultDropdown` doesn't quite describe the purpose of the prop